challenges/helpers.py

```diff
@@ -0,0 +1,23 @@
+from rest_framework.exceptions import NotFound
+from challenges.registry import CHALLENGES, CHALLENGE_SERIALIZERS
+
+
+def get_challenge(name) -> type:
+    challenge = dict(CHALLENGES).get(name, None)
+    if challenge is None:
+        raise NotFound
+    return challenge
+
+
+def get_challenge_serializer(name) -> type:
```

Comment:
Serializers and models should be independent. The is just a shorthand method to be used in views. It throws 404 Not Found if the serializers was not found